### PR TITLE
change registration to close

### PR DESCRIPTION
**Pull request by dquach217**
Close registration

### CODE REVIEW

**Comment by sman591** [comment]
Also the homepage should probably explicitly state that registration has closed, or people will visit and be confused / frustrated / complain

**Comment by dquach217** [comment]
@sman591 made some changes

File: app/views/pages/index.html.haml [hunk 1]
```diff
@@ -46,7 +47,6 @@
       - elsif Rails.configuration.hackathon['registration_is_open']
         %div
           = btn_link_to "Register Now", new_user_registration_path
-          = btn_link_to "Volunteer", volunteer_link
```
Comment (sman591) [inline app/views/pages/index.html.haml]: nit: probably want to keep the volunteer link here for the future

Comment (dquach217) [reply app/views/pages/index.html.haml]: I just commented it out for the future

**Pull request merged**